geode-core/src/main/java/org/apache/geode/internal/cache/control/HeapMemoryMonitor.java

```diff
@@ -844,4 +842,13 @@ public static void setTestDisableMemoryUpdates(final boolean newTestDisableMemor
   public static void setTestBytesUsedForThresholdSet(final long newTestBytesUsedForThresholdSet) {
     testBytesUsedForThresholdSet = newTestBytesUsedForThresholdSet;
   }
+
+  private String getCriticalThresholdLogMessage(MemoryEvent event, String eventOrigin,
```

Comment:
nitpick, it's generating a log string, maybe name could be createCriticalThresholdLogMessage?

Reply:
Sure, works for me